shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Pretty.hs

```diff
@@ -1211,3 +1301,37 @@ atWidth n a pp = do
   let doc = pp a
   putDocW n doc
   putStrLn ""
+
+-- =========================================================
+-- Wrap a type: t, with a PrettyA instance, so that the
+-- prettyA instance is used to Show (Nice t).
+
+newtype Nice t = Nice {unNice :: t}
```

Comment:
[minor] Same comment I've made in a bunch of places - it's much more helpful to have this comment as haddock on the type. Would also be good to clarify that this is intended as a `DerivingVia` helper.

Reply:
Done.